pkg/metric/local_stat_collector.go

```diff
@@ -0,0 +1,45 @@
+package metric
+
+import (
+	"github.com/kubernetes-sigs/alibaba-cloud-csi-driver/pkg/local/server"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	vgFreeBytesDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(nodeNamespace, volumeSubSystem, "vg_free_bytes"),
+		"Total amount of free bytes in LVM volume group",
+		[]string{"vg_name"}, nil,
+	)
+	vgSizeBytesDesc = prometheus.NewDesc(
+		prometheus.BuildFQName(nodeNamespace, volumeSubSystem, "vg_size_bytes"),
+		"Total size of LVM volume group in bytes",
+		[]string{"vg_name"}, nil,
+	)
+)
+
+func init() {
+	registerCollector("local_stat", NewLocalStatCollector)
```

Comment:
we should not enable local_stat if localplugin is not enabled

Reply:
let's do this in another PR.